.devcontainer/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:22.04
```

Comment:
After testing some stuff on mobile, also suggest adding `--platform=linux/amd64` to be compatible with our prod machines running on x86 arch

Reply:
I'm assuming this will fix dependency resolution? Sounds good to me!